### PR TITLE
Notebook 2

### PR DESCRIPTION
**Pull request by marco-2023**
Second notebook NCI water dimer added .

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+.. code-block:: bash
+
+   git clone https://github.com/theochem/AtomDB.git
+
+
+Then installation via pip can be done by going into the directory where AtomDB was downloaded to
+and running,
+
+
+.. code-block:: bash
```
Comment (gabrielasd) [inline docs/source/installation.rst:41]: This is just a recomentadion and it is mainly pertinent to people who may want to contribute to the package:
And section for installation inside conda venv
Add code snippet to show installation in editable mode 

For example I like the Istallation instructions section from [ChemTools](https://chemtools.org/usr_doc_installation.html)

File: docs/source/installation.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Building Documentation
+======================
+
+The documentation can be built locally by running the following commands in the doc directory,
+
+
+.. code-block:: bash
+
+    make html
```
Comment (gabrielasd) [inline docs/source/installation.rst:123]: Another suggestion, include command to change from root directory of package to doc directory:
`cd docs && make html`

File: docs/source/atomdb_cite.bib [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@Article{atomdb,
+    author = {S{\'a}nchez D{\'i}az, Gabriela and Richer, Michelle and Tehrani, Alireza and
```
Comment (gabrielasd) [inline docs/source/atomdb_cite.bib:2]: author = {S{\'a}nchez D{\'i}az, Gabriela and Richer, Michelle and Mart\'{i}nez Gonz\'{a}lez, Marco and van Zyl, Maximilian and Pujal, Leila and Tehrani, Alireza and Bianchi, Julianna and Ayers, Paul W. and Heidar-Zadeh, Farnaz},

**Review by gabrielasd — APPROVED** [review]
Looks great!

File: docs/source/installation.rst (hunk 2 above)
Comment (marco-2023) [reply docs/source/installation.rst:123]: In this case, we said that the command should be run from the doc directory.

File: docs/source/atomdb_cite.bib (hunk 3 above)
Comment (marco-2023) [reply docs/source/atomdb_cite.bib:2]: Done!

File: docs/source/installation.rst (hunk 1 above)
Comment (marco-2023) [reply docs/source/installation.rst:41]: I added the editable installation.

**Comment by marco-2023** [comment]
Thanks @gabrielasd for your suggestions!  I followed the straightforward ones. We can always do a new review a the end.

**Pull request merged**